staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_util.go

```diff
@@ -265,7 +243,7 @@ func getDatastoresForZone(ctx context.Context, dc *vclib.Datacenter, nodeManager
 
 		for _, host := range hosts {
 			var hostSystemMo mo.HostSystem
-			err = host.Properties(ctx, host.Reference(), []string{"datastore"}, &hostSystemMo)
+			err = host.Properties(ctx, host.Reference(), []string{"datastore", "parent"}, &hostSystemMo)
```

Comment:
This parent might not be needed at this step since you are using Ancestors() to find the parent dc.

Reply:
Done. Thanks for the review.